File: UVA/UVA-11727.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n;
    cin>>n;
    for(int i=1;i<=n;i++)
    {
        int arr[3];
        cin>>arr[0]>>arr[1]>>arr[2];
        sort(arr, arr+3);
        cout<<"Case "<<i<<": "<<arr[1]<<endl;
    }

    return 0;
}

File: UVA/UVA-11727.java
import java.util.Scanner;
import java.util.Arrays;
public class Main
{
    public static void main (String[]args)
    {
        Scanner puttingvaluesinside=new Scanner(System.in);
        for(int a=puttingvaluesinside.nextInt(), b=1;a>0;a--, b++)
        {
            int arr[]=
            {
                puttingvaluesinside.nextInt(),
                puttingvaluesinside.nextInt(),
                puttingvaluesinside.nextInt()
            };
            Arrays.sort(arr);
            System.out.println("Case "+b+": "+arr[1]);
        }
    }
}